tests/suites/os/suite.js

```diff
@@ -76,7 +76,8 @@ module.exports = {
 			this.suite.options.balena.organization, 
 			join(homedir(), 'id')
 		);
-		const cloud = new Balena(this.suite.options.balena.apiUrl, this.getLogger());
+
+		const cloud = new Balena(this.suite.options?.balena?.apiUrl, this.getLogger());
```

Comment:
Won't this throw if apiUrl is unset?

Reply:
@klutchell I'm not sure what the _intended_ result is, but I just tested with no `apiKey` or `apiUrl`, and the unmanaged OS suite completes without a problem. I could add a conditional that would leave `cloud` undefined if the parameters are unspecified, instead of passing them in as undefined.